addons/point_of_sale/models/pos_order.py

```diff
@@ -467,14 +467,25 @@ def action_pos_order_paid(self):
     def _prepare_invoice_vals(self):
         self.ensure_one()
         timezone = pytz.timezone(self._context.get('tz') or self.env.user.tz or 'UTC')
+        note = self.note or ''
```

Comment:
@trj-odoo most of the function really looks like a copy paste from the code in sale, can't we create a funnction in account module in 'account_move.py' file called "_get_default_terms_and_condition" and call it here and in the function "_default_note" of the sale order?

Reply:
yeah could do

Reply:
Nvm, this PR is not relevant anymore. https://github.com/odoo/odoo/pull/73121 changed how T&C are added in the invoice. I need to adapt the code